test/atmosphere_model_unit.jl

```diff
@@ -1,9 +1,27 @@
 using Breeze
 using GPUArraysCore: @allowscalar
 using Oceananigans
+using Oceananigans.Diagnostics: erroring_NaNChecker!
 using Oceananigans.Operators: ℑzᵃᵃᶠ
 using Test
 
+function run_nan_checker_test(arch; erroring)
+    grid = RectilinearGrid(arch, size=(4, 2, 1), extent=(1, 1, 1))
+    model = AtmosphereModel(grid)
+    simulation = Simulation(model, Δt=1, stop_iteration=2)
+    @allowscalar model.momentum.ρu[1, 1, 1] = NaN
+    erroring && erroring_NaNChecker!(simulation)
+
+    if erroring
+        @test_throws ErrorException run!(simulation)
+    else
+        run!(simulation)
+        @show @test model.clock.iteration == 1 # simulation stopped after one iteration
```

Comment:
I presume this `@show` is a leftover?

Reply:
Yeah! Thanks for catching this. Removed in #294.